website/docs/r/tg_gateway.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "ibm"
+page_title: "IBM : tg_gateway"
+sidebar_current: "docs-ibm-resource-tg-gateway"
+description: |-
+  Manages IBM Transit Gateway.
+---
+
+# ibm\_dl_gateway
```

Comment:
should this be `ibm\_tg_gateway`?

Reply:
Hi Frederic.. Thanks, Yes, it should be changed, will change it.